### DISCUSSION TITLE
Multiple dns names?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5877] [Q&A] [answered]
Hi, MC has a regular dns-name and it's working with. IS it possible to add an aliasname and use this aliasname too?
i just try simple to open MC in brwoser with the aliasname and i can login, but i can't do anything furthermore, there is a blank screen with "invalid origin in HTTP request,"....
Any ideas?

**Comment by si458** [comment, marked as answer]
The invalid origin is a security feature to make sure u access meshcentral from the correct dns name
Plz read the post here about setting it up https://github.com/Ylianst/MeshCentral/issues/5826#issuecomment-1951914601

**Comment by simpsonetti** [comment]
Thanks a lot.
